client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -102,35 +107,45 @@ const Task = ( {
 	};
 
 	return (
-		<ActionPanel className="task">
+		<ActionPanel className={ classNames( 'task', taskId ) }>
```

Comment:
Would be handy to have the taskId as a class here to allow for task specific css targeting, eg. @jancavan [was hoping a gradient could be applied to the block editor image in the editor deprecation tasks](className={ classNames( 'task', taskId ) }), but this isn't possible unless we can target just tasks of that type.